### ISSUE TITLE
Unable to launch the default code to test the sdk

### ISSUE THREAD

**Issue by descampsk** [mindee/mindee-api-nodejs#96]
### Prerequisites

Put an X between the brackets on this line if you have done all of the following:

* [x] Reproduced the problem or exposed a new need
* [x] Checked the github existing issues

### Steps to Reproduce

1. Create a new project `npm init`
2. Use node 16 `nvm use 16`
3. Install mindee `npm i mindee --save`
4. Create index.js
```js
import { Client, InvoiceResponse } from "mindee";

// Init a new client
const mindeeClient = new Client({apiKey: "my-api-key"});

// Load a file from disk and parse it
const invoiceResponse = mindeeClient.docFromPath("/path/to/the/invoice.pdf")
  .parse(InvoiceResponse);

// Print a brief summary of the parsed data
invoiceResponse.then((resp) => {
  console.log(resp.document);
});
```
5. Launch `node index.js`

**Expected behavior:**

Should receive a response without error

**Actual behavior:**

First error : 
```
(node:1952) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
/mnt/c/Programmation/mindee/index.js:1
import { Client, InvoiceResponse } from "mindee";
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at Object.compileFunction (node:vm:360:18)
    at wrapSafe (node:internal/modules/cjs/loader:1055:15)
    at Module._compile (node:internal/modules/cjs/loader:1090:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
    at Module.load (node:internal/modules/cjs/loader:1004:32)
    at Function.Module._load (node:internal/modules/cjs/loader:839:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:17:47
```

If I use a module which should not be mandatory:
```
node:internal/errors:477
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find package '/mnt/c/Programmation/mindee/node_modules/mindee/' imported from /mnt/c/Programmation/mindee/index.js    
    at new NodeError (node:internal/errors:387:5)
    at legacyMainResolve (node:internal/modules/esm/resolve:341:9)
    at packageResolve (node:internal/modules/esm/resolve:938:14)
    at moduleResolve (node:internal/modules/esm/resolve:1000:20)
    at defaultResolve (node:internal/modules/esm/resolve:1214:11)
    at nextResolve (node:internal/modules/esm/loader:165:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:844:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:431:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:76:40)
    at link (node:internal/modules/esm/module_job:75:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

**Reproduces how often:**

100%

### Versions
OS : WSL 
$ npm -v
8.15.0
$ node -v
v16.17.1

**Comment by ianardee** [comment]
Thanks a lot for reporting this issue.

The problem should be fixed with **version 2.0.1**.

The sample code has been updated as follows:
```js
const mindee = require("mindee");
// for TS or modules:
// import * as mindee from "mindee";

// Init a new client
const mindeeClient = new mindee.Client({apiKey: "my-api-key"});

// Load a file from disk and parse it
const invoiceResponse = mindeeClient.docFromPath("/path/to/the/invoice.pdf")
    .parse(mindee.InvoiceResponse);

// Print a brief summary of the parsed data
invoiceResponse.then((resp) => {
    console.log(resp.document);
});
```

Please let me know if it works for you after updating to v2.0.1.

Thanks!

**Comment by ianardee** [comment]
Other users have confirmed the fix, but feel free to re-open if there is still a problem.
